contrib/dist/linux/README

```diff
@@ -36,8 +36,53 @@ one or more RPMs from this tarball:
    - openmpi-devel
    - openmpi-docs
 
-The prefix, target architecture, and choice of RPM(s) to build are all
-currently hard-coded in the beginning of the buildrpm.sh script.
+The folowing arguments could be used to affect script behaviour.
+Please, do NOT set the same settings with parameters and config vars.
+
+-b
+   If you specify this option, all-in-one binary RPM would be created
+   instead of source RPM (SRPM). This option is necessary for other
+   parameters to work, as they won't be influence SRPM package.
```

Comment:
I have fixed the description. It should be a bit better now.

Basically, all the parameters influence only build of the  binary RPM. If user build only Source RPM, those parameters won't have any effect as they are not modifying specfile.